docs/building-apps/wallet/sep6.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Programmatic Deposit and Withdrawal
+sidebar_position: 60
+---
+
+import { LanguageSpecific } from "@site/src/components/LanguageSpecific";
+
+import { WalletCodeExample as CodeExample } from "@site/src/components/WalletCodeExample";
+import Header from "./component/header.mdx";
+
+<Header WIPLangs={["kotlin", "dart"]} />
+
+The [SEP-6] standard defines a way for anchors and wallets to interact on behalf of users. Wallets use this standard to facilitate exchanges between on-chain assets (such as stablecoins) and off-chain assets (such as fiat, or other network assets such as BTC).
+
+Please note, this is for _programmatic_ deposits and withdrawals. For hosted deposits and withdrawals, where the anchor interacts with wallets interactively using a popup, please see [Hosted Deposit and Withdrawal](/docs/building-apps/wallet/sep24).
+
+## Get Anchor Information
+
+Let's start with creating a sep6 object, which we'll use for all SEP-6 interactions:
+
+<CodeExample>
+
+```typescript
+const sep6 = await anchor.sep6();
+```
+
+</CodeExample>
+
+First, let's get information about the anchor's support for [SEP-6]. This request doesn't require authentication, and will return generic info, such as supported currencies, and features supported by the anchor. You can get a full list of returned fields in the [SEP-6 specification](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#info).
+
+<CodeExample>
+
+```typescript
+const info = await sep6.info();
+```
+
+</CodeExample>
+
+## Start a deposit
+
+Before getting started, make sure you have connected to the anchor and received an authentication token, as described in the [Stellar Authentication] wallet guide. We will use `authToken` in the examples below as the [SEP-10] authentication token, obtained earlier.
+
+To initiate an operation, we need to know an asset. You can hardcode this. Ensure it is one that is included in the info response above.
+
+:::info
+
+Before starting with the deposit flow, make sure that the user account has [established a trustline](/docs/building-apps/wallet/stellar#modify-assets-trustlines) for the asset you are working with.
+
+:::
+
+Let's start with a basic deposit. We will use `account` to represent our account's public key.
+
+<CodeExample>
+
+```typescript
+const deposit = await sep6.deposit({
+  authToken,
+  params: {
+    asset_code,
+    account,
+  },
+});
+```
+
+</CodeExample>
+
+There are several kinds of responses, depending on if the anchor needs more information. All deposits and withdrawals will have these same response types:
```

Comment:
Can you add code example of how to handle different response?

Reply:
good idea, hows this https://github.com/stellar/stellar-docs/pull/283/commits/1fb446fbafba1b90bbeba3cbb1d6f2c74685cff0

for the other 403 response I don't think it's worth adding a code example, there's not a standard way to handle